src/main/java/com/hunllefhelper/HunllefHelperConfig.java

```diff
@@ -9,13 +11,13 @@ public interface HunllefHelperConfig extends Config
 {
 	@ConfigItem(
 		position = 1,
-		keyName = "autoHide",
-		name = "Automatically hide",
-		description = "If checked, only show the plugin panel inside The Gauntlet"
+		keyName = "visibilityMode",
+		name = "Visibility Mode",
+		description = "Determines when the plugin panel is shown."
```

Comment:
I think it would look nicer if this configuration is a bit more intuitive. 
```suggestion
		keyName = "panelVisibility",
		name = "Show plugin panel",
		description = "Determines when the plugin panel is shown."
```